datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -439,6 +441,15 @@ func datadogSecurityMonitoringRuleSchema() map[string]*schema.Schema {
 			Description: "The rule type.",
 			Default:     "log_detection",
 		},
+		"validate": {
+			Description: "Whether or not to validate the SLO.",
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Could you update the comment.

Reply:
Done.